lib/rails/generators/intercom/config/intercom.rb.erb

```diff
@@ -21,8 +21,7 @@ IntercomRails.config do |config|
   <%- end -%>
 
   # == Intercom API Key
-  # This is required for some Intercom rake tasks like importing your users;
-  # you can generate one at https://app.intercom.io/apps/api_keys.
+  # You can generate one at https://app.intercom.io/apps/api_keys.
```

Comment:
Yeh I don't know. The description certainly sounds like they're not... So you reckon it's safe to remove?